dotnet-desktop-guide/net/winforms/printing/overview.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "PrintDialog Component Overview"
+ms.description: "Learn about print related settings in Windows-based applications."
+ms.date: "04/13/2022"
+f1_keywords:
+  - "PrintDialog"
+helpviewer_keywords:
+  - "Print dialog box [Windows Forms], displaying"
+  - "PrintDialog component [Windows Forms], about PrintDialog component"
+ms.custom: devdivchpfy22
+---
+
+# PrintDialog Component Overview (Windows Forms)
```

Comment:
First header should have .NET in it to distinguish it from the other docsets
```suggestion
# PrintDialog Component Overview (Windows Forms .NET)
```

Reply:
done